css/index.css

```diff
@@ -1,3 +1,453 @@
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+html, body {
+    font-family: 'Titillium Web', sans-serif;
+    height: 100%;
+}
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+/* Your code starts here! */
+body {
+    align-items: center;
+	display: flex;
+	flex-direction: column;
+	height: auto;
+}
+body .container {
+    width: 880px;
+}
+/*TOP NAVIGATION*/
+header {
+    margin-top: 16px;
+}
+header nav {
+    display: flex;
+    width: 100%;
+    justify-content: center;
+}
+header nav .links {
+    display: flex;
+    justify-content: space-between;
+    flex: 1;
+    align-items: center;
+    height: 60px;
+}
+header nav .links .link {
+    text-decoration: none;
+    color: rgb(174,174,174);
+    margin-top: 3px;
+}
+header nav .logo {
+    display: block;
+    background-image: url("../img/logo.png");
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center center;
+    width: 183px;
+    height: 60px;
+    margin-left: 90px;
+}
+/*HEADER TEXT AND IMAGE*/
+#header-banner {
+	align-items: center;
+	display: flex;
+	justify-content: center;
+	margin-top: 60px;
+}
+#header-banner #header-text {
+	align-items: center;
+	display: flex;
+	flex-direction: column;
+	height: 344px;
+		width: 480px;
+	margin-top: 35px;
+}
+#header-banner h1 {
+	font-size: 72px;
+	line-height: 73px;
+	margin-top: 13px;
+	margin-bottom: 0px;
+    margin-left: 74px;
+    text-align: center;
+}
+#header-banner a {
+	border: 1px solid black;
+	color: black;
+	text-align: center;
+	font-size: 17px;
+	height: 32px;
+		width: 169px;
+    text-decoration: none;
+    margin: 24px 0 0 74px;
+    padding: 7px 0 0 1px;
+}
+#header-banner figure {
+	background-image: url(../img/header-img.png);
+	background-size: contain;
+	background-repeat: no-repeat;
+	height: 385px;
+	width: 385px;
+	margin: 0;
+	margin-left: 29px;
+}
+/*MAIN */
```

Comment:
good use of comments.. but main of what? Try to be more specific as if someone (like me) who might not be familiar with your code.